workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/CrawlerActionNode.java

```diff
@@ -74,19 +72,12 @@ public String getXML() {
      * @return String containing main class to be appended to workflow string
      */
     public String getClassName(Integer pid, String configGroup) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "pid,configGroup". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)